docs/NEW_LANGUAGE_FEATURE_INTEGRATION_PROCESS.md

```diff
@@ -0,0 +1,31 @@
+# Check Update Procedure
```

Comment:
Let's use what is in the issue: `New Language Feature Check Integration Process [WIP]` with the clear `[WIP]` tag at the end.